okapi-core/src/test/java/org/folio/okapi/ProxyTest.java

```diff
@@ -4196,6 +4196,87 @@ private void setupBasicModule(String tenant, String moduleId, String tenantPermi
         .then().statusCode(200).log().ifValidationFails();
   }
 
+  @Test
+  public void testImportModules(TestContext context) {
+    RestAssuredClient c;
+
+    c = api.createRestAssured3();
+    given()
+        .header("Content-Type", "application/json")
+        .body("{\"id\":").post("/_/proxy/import/modules")
+        .then().statusCode(400);
+
+    c = api.createRestAssured3();
+    c.given()
+        .header("Content-Type", "application/json")
+        .body("[]").post("/_/proxy/import/modules")
+        .then().statusCode(204);
+    Assert.assertTrue("raml: " + c.getLastReport().toString(),
+        c.getLastReport().isEmpty());
+
+    ModuleDescriptor mdA = new ModuleDescriptor();
+    mdA.setId("moduleA-1.0.0");
+    {
+      RoutingEntry[] routingEntries = new RoutingEntry[1];
+      RoutingEntry routingEntry = routingEntries[0] = new RoutingEntry();
+      routingEntry.setPathPattern("/a");
+      routingEntry.setMethods(new String[]{"GET"});
+
+      InterfaceDescriptor[] interfaceDescriptors = new InterfaceDescriptor[1];
+      InterfaceDescriptor interfaceDescriptor = interfaceDescriptors[0] = new InterfaceDescriptor();
+      interfaceDescriptor.setId("intA");
+      interfaceDescriptor.setVersion("1.0");
+      interfaceDescriptor.setHandlers(routingEntries);
+      mdA.setProvides(interfaceDescriptors);
+    }
```

Comment:
Such code blocks can be avoided if the classes provide methods/constructors that reduce noise in the test setup.
```
mdA.setProvides("intA", "1.0", new RoutingEntry("/a", "GET"));
```

```
public RoutingEntry(String pathPattern, String... httpMethods)
```